apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -179,6 +193,20 @@
 
 		<!-- loading indicator to replace the menu -->
 		<div v-else class="icon-loading-small sharing-entry__loading" />
+
+		<!-- Modal to open whenever we have a QR code -->
+		<NcDialog size="normal"
+			:open.sync="showQRCode"
+			:name="title"
+			:closeOnClickOutside="true"
```

Comment:
To follow style:
```suggestion
			:close-on-click-outside="true"
```

Or if also follow the HTML-like style of Vue templates:
```suggestion
			close-on-click-outside
```